test/features/features/composer/recipient_composer_widget_test.dart

```diff
@@ -262,5 +264,99 @@ void main() {
+          prefix: prefix,
+          listEmailAddress: listEmailAddress,
+          imagePaths: imagePaths,
+          maxWidth: 360,
+          expandMode: ExpandMode.COLLAPSE,
+          keyTagEditor: keyEmailTagEditor,
+        ),
+      );
+
+      await tester.pumpWidget(widget);
+
+      await tester.pumpAndSettle();
+
+      final prefixRecipientComposerWidgetFinder = find.byKey(Key('prefix_${prefix.name}_recipient_composer_widget'));
+      final recipientTagItemWidgetFinder = find.byKey(Key('recipient_tag_item_${prefix.name}_0'));
+
+      final Size prefixRecipientComposerWidgetSize = tester.getSize(prefixRecipientComposerWidgetFinder);
+      final Size recipientTagItemWidgetSize = tester.getSize(recipientTagItemWidgetFinder);
+
+      log('recipient_composer_widget_test::main: PrefixLabelSize = $prefixRecipientComposerWidgetSize | TagSize = $recipientTagItemWidgetSize');
+
+      expect(prefixRecipientComposerWidgetFinder, findsOneWidget);
+      expect(recipientTagItemWidgetFinder, findsOneWidget);
+      expect(
+        prefixRecipientComposerWidgetSize.width + recipientTagItemWidgetSize.width,
+        lessThanOrEqualTo(360)
+      );
+
+      final labelRecipientTagItemWidgetFinder = find.byKey(Key('label_recipient_tag_item_${prefix.name}_0'));
+      final deleteIconRecipientTagItemWidgetFinder = find.byKey(Key('delete_icon_recipient_tag_item_${prefix.name}_0'));
+      final avatarIconRecipientTagItemWidgetFinder = find.byKey(Key('avatar_icon_recipient_tag_item_${prefix.name}_0'));
+      final counterRecipientTagItemWidgetFinder = find.byKey(Key('counter_recipient_tag_item_${prefix.name}_0'));
+
+      final Size labelRecipientTagItemWidgetSize = tester.getSize(labelRecipientTagItemWidgetFinder);
+      final Size deleteIconRecipientTagItemWidgetSize = tester.getSize(deleteIconRecipientTagItemWidgetFinder);
+      final Size avatarIconRecipientTagItemWidgetSize = tester.getSize(avatarIconRecipientTagItemWidgetFinder);
+      final Size counterRecipientTagItemWidgetSize = tester.getSize(counterRecipientTagItemWidgetFinder);
+
+      log('recipient_composer_widget_test::main: LabelTagSize = $labelRecipientTagItemWidgetSize | DeleteIconTagSize = $deleteIconRecipientTagItemWidgetSize | AvatarIconTagSize = $avatarIconRecipientTagItemWidgetSize | CounterTagSize = $counterRecipientTagItemWidgetSize');
+
+      expect(labelRecipientTagItemWidgetFinder, findsOneWidget);
+      expect(deleteIconRecipientTagItemWidgetFinder, findsOneWidget);
+      expect(avatarIconRecipientTagItemWidgetFinder, findsOneWidget);
+      expect(counterRecipientTagItemWidgetFinder, findsOneWidget);
+
+      final totalSizeOfAllComponents = labelRecipientTagItemWidgetSize.width +
+        deleteIconRecipientTagItemWidgetSize.width +
+        avatarIconRecipientTagItemWidgetSize.width;
+        counterRecipientTagItemWidgetSize.width;
+
+      expect(
+        totalSizeOfAllComponents,
+        lessThanOrEqualTo(recipientTagItemWidgetSize.width)
```

Comment:
IMO, it should be `lessThan` because we not sum the width of `drop down` button.

Reply:
OK, I agree it's smaller than right because we haven't taken into account the `Space/Padding/Margin` components yet. As for the `drop down` button, it has nothing to do with this UI, it belongs to another UI.